src/Core/src/Platform/iOS/WindowExtensions.cs

```diff
@@ -36,9 +37,11 @@ internal static void UpdateHeight(this UIWindow platformWindow, IWindow window)
 
 		internal static void UpdateCoordinates(this UIWindow platformWindow, IWindow window)
 		{
-			if (OperatingSystem.IsMacCatalyst() && OperatingSystem.IsIOSVersionAtLeast(16) && platformWindow.WindowScene is { } windowScene)
+			if (OperatingSystem.IsMacCatalyst() && OperatingSystem.IsIOSVersionAtLeast(16) &&
```

Comment:
This platform check looks weird: you're checking for Mac Catalyst, then verifying the iOS version... I know why it works (because Mac Catalyst is a sub-platform of iOS), but IMHO this would read better if calling IsMacCatalystVersionAtLeast instead of IsIOSVersionAtLeast.